app/templates/direct-award/view-project.html

```diff
@@ -18,7 +19,11 @@
           "link": url_for('direct_award.saved_search_overview', framework_family=framework.framework),
           "label": "Your saved searches"
       }
-    ]
+    ] if project else [
+      {
+          "label": "Find cloud hosting, software and support"
```

Comment:
It feels weird that this template is sometimes used in one breadcrumbs style (i.e. with the last item being a link) and the other (i.e. with no link, like here when there's no project)!

Maybe we can just add the current page name here (with no link). And seeing as it's repeated in two other places (in the title and in the heading), then maybe it could then be a variable?

Reply:
I don't *think* variables can span across jinja `{% block%}`s otherwise I would have done so.

Reply:
re other stuff, doing it like this is just what kuba said in the ticket, so...

Reply:
Re variables, it definitely does work - see https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/796/files#diff-65331f8ea2a95fe2321e350a64964497R3 :-)

Reply:
Oh! Think I was assuming because of the django template lang. Wonder how it copes with the scope ambiguity.